include/small/matras.h

```diff
@@ -157,7 +158,7 @@ struct matras_view {
 	/* root extent of the view */
 	void *root;
 	/* block count in the view */
-	matras_id_t block_count;
+	size_t block_count;
```

Comment:
Doubling the size of this variable just to accommodate +1 looks excessive. Would it be possible to rename this variable to `max_block_id` and leave its type `uint32_t`?